packages/core/src/components/sections/ProductGallery/section.module.scss

```
@@ -13,6 +14,25 @@
       }
     }
 
+    [data-fs-search-loading] {
+
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+
+      h1 {
+        font-size: var(--fs-text-size-4);
+
+        @include media(">=tablet") { font-size: var(--fs-text-size-5); padding-top: var(--fs-spacing-4); }
```

just trying to keep the pattern
```suggestion
        @include media(">=tablet") { 
          font-size: var(--fs-text-size-5); 
          padding-top: var(--fs-spacing-4); 
        }
```

---

I  executed the stylelint now and it keep the same code